setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+with open(path.join(here, 'README.md')) as f:
+    description = f.read()
+
+setup(
+    name='cinch',
+    version='0.1.6',
```

Comment:
I feel like it is. We're declaring it pre-alpha, it's pre-1.0 versioning. We haven't even made a release yet. And I've just done a lot of iterating on what I would call a 0.1 release here. The .6 came up just as I was testing updating and upgrading it in-place. Our first "release" should probably be called 0.2.0 just to keep things sane.